misk-hibernate/src/main/kotlin/misk/hibernate/ProtoColumn.kt

```diff
@@ -0,0 +1,11 @@
+package misk.hibernate
+
+/**
+ * Put this on a column field for a class generated by Wire to get Hibernate to persist
+ * with protobuf serialization.
+ *
+ * It is an error to put this annotation on a mutable field. We don't defensively copy these when
```

Comment:
I don't follow this comment. it's obviously a copy of the data that is in the actual database

Reply:
Protos are typically immutable. For JSON this comment was more appropriate where you could assign a thing and then mutate it